clar2wasm/src/tools.rs

```diff
@@ -400,6 +400,27 @@ pub fn crosscheck_compare_only_advancing_tip(snippet: &str, count: u32) {
     );
 }
 
+pub fn crosscheck_at(snippet: &str, expected: Result<Option<Value>, ()>, epoch: StacksEpochId) {
```

Comment:
Super weird name. I would need to read the source code to understand that this function specificity is to take a `StacksEpochId`.

Reply:
Isn't it what `interpret_at` and `evaluate_at` doing? I'll rename that though.

Reply:
Done

Reply:
I honestly had no idea that's what the `at` meant for those functions! :o